NIPTool/load/batch.py

```diff
@@ -1,54 +1,65 @@
 import logging
 from NIPTool.build.document import build_sample, build_batch
-from NIPTool.parse.batch import parse_batch_file
 from NIPTool.models.validation import requiered_fields
+from NIPTool.exeptions import MissingResultsError
+from pathlib import Path
+
+import json
 
 
 LOG = logging.getLogger(__name__)
 
 
 def check_requiered_fields(document):
-    """Check if document keys contain requiered fields"""
+    """Check if document keys contain required fields"""
 
     if not set(requiered_fields).issubset(set(document.keys())):
         LOG.info(f"Could not add document {document}. Requiered fields missing.")
         return False
     return True
 
 
-def load_result_file(adapter, batch_data: list, project_name: str) -> None:
+def load_batch(adapter, batch_data: dict, request_data: dict) -> None:
     """Function to load data from fluffy result file. 
     Raises:
```

Comment:
The exception is not raised in the code of the function. Perhaps could be handled here?